io.mjml.mjml-app.yaml

```
@@ -0,0 +1,49 @@
+app-id: io.mjml.mjml-app
+base: io.atom.electron.BaseApp
```

You should be using the newest electron baseApp instead
```suggestion
base: org.electronjs.Electron2.BaseApp
```

---

I had to revert to `io.atom.electron.BaseApp`, as `org.electronjs.Electron2.BaseApp` throws an error on statup that libgtk-x11-2.0.so is missing.

---

@janus-reith Oh, didn't knew that you were not using the latest electron version. They have switched to gtk3 that's why the library is missing on the latest baseApp.